exchanges/huobi/huobi_websocket.go

```diff
@@ -299,45 +321,30 @@ func (h *HUOBI) wsHandleData(respRaw []byte) error {
 		if err != nil {
 			return err
 		}
-	case strings.Contains(init.Rep, "kline"):
```

Comment:
This was removed due to no subscription data coming through, no WebSocket functionality to test it and should be utilizing SendMessageReturnResponse()

I will add a note to add that functionality so it can be used wrapper side.  

Side note WsResponse has `json "rep"` tag when docs have `"req": "market.$symbol.kline.$period",`